include/psp2kern/kernel/intrmgr.h

```diff
@@ -41,7 +41,7 @@ typedef struct SceKernelIntrOptParam {
 } SceKernelIntrOptParam;
 
 int ksceKernelRegisterIntrHandler(int intr_code, const char *name, int interrupt_type,
-	SceKernelIntrHandler *handler, void *userCtx, int priority, int targetcpu, SceKernelIntrOptParam *opt);
+	SceKernelIntrHandler handler, void *userCtx, int priority, int targetcpu, SceKernelIntrOptParam *opt);
```

Comment:
could you change these?
- `userCtx` to `user_ctx`
- `targetcpu` to `target_cpu`

Reply:
done